src/components/skip-link/_skip-link.scss

```diff
@@ -11,5 +11,11 @@
 
     display: block;
     padding: govuk-spacing(2) govuk-spacing(3);
+
+    // Add safe area (e.g. iPhone X, Google Pixel 3)
+    @supports (padding: env(safe-area-inset-left)) {
```

Comment:
Does this change impact the padding when the phone is portrait?

Reply:
Realised this is what the 'max' is for 👍

Reply:
Yeah, although technically both portrait and landscape could have an inset.

For example, a circular Google Wear OS watch screen. It could be offset all around to avoid going to close to the edges of the circle.

Reply:
More info on that here: https://drafts.csswg.org/css-round-display/

Reply:
Resolving this, all questions answered